### PR TITLE
feat(navigation): improve children visualization

### PR DESCRIPTION
**Pull request by rbarilani**
Improving children visualisation by adding some left "margin" to children list and highlight also current item ancestors (if any).

### CODE REVIEW

File: lib/browser/navigation/containers.jsx [hunk 1]
```diff
@@ -52,6 +54,45 @@ class Navigation extends React.Component {
...
+    const {navigation} = Object.assign({}, { navigation: {} }, this.props.data);
+    const items = navigation.main || [];
+
+    (function recurse (items, parent) {
+      items.forEach((item) => {
+        // add parent methods
+        item.parent = () => { return parent; };
+        item.hasParent = () => {
+          const p = item.parent();
+          return typeof p !== undefined;
```
Comment (bhaskarmelkani) [inline lib/browser/navigation/containers.jsx]: Guess you meant  `return typeof p !== "undefined";` or we can also do `!!p`;

**Comment by rbarilani** [comment]
@bhaskarmelkani fixed!

**Comment by bhaskarmelkani** [comment]
👍

**Comment by rbarilani** [comment]
👍

**Pull request merged**